### PR TITLE
[CPP Onboarding] Remove feature flag for In-Person Payments Onboarding

### LINKED ISSUE #4838: [CPP Onboarding] Remove feature flag
Remove the cardPresentOnboarding feature flag

### PR DESCRIPTION
**Pull request by ctarda**
Closes #4838 

This PR removes the feature flag for onboarding into In-Person Payments. The feature is still available only for stores gated into the beta.

| Before | After |
| ---- | ---- |
|  |  |

## Changes
* Remove the feature flag.Bye!

## How to test
* On a store that is gated into In-Person Payments, navigate to Settings. Notice the previous "Manage Card Reader" entry is gone, and now it should read "In-Person Payments"
* Test that it is still possible to connect to a card reader
* Switch to a Store that is not gated into In-Person Payments. Navigate to Settings Check that "In-Person Payments" is not visible

Update release notes:
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift [hunk 1]
```diff
@@ -209,9 +209,6 @@ private extension SettingsViewController {
     private var storeSettingsRows: [Row] {
         var result = [Row]()
         if canCollectPayments {
-            result.append(.cardReadersV2)
```
Comment (allendav) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift:212]: We don't want to delete this

Comment (ctarda) [reply WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift:212]: Yes, we do 😄 . This is part of the new "In Person Payments menu item"

Comment (allendav) [reply WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift:212]: Ah, so the connection flow will still be served by that. I see.

I withdraw my complaint. Carry on.

**Review by allendav — APPROVED** [review]
Tests well in light mode and dark mode on iPhone 12 Pro Max :shipit:

**Review by allendav — APPROVED** [review]
> Remove other traces of the cardReaderV2 section

Continues to test well in light mode and dark mode on iPhone 12 Pro Max :shipit:
Was able to connect to reader.
Noted that tracks event made it over too.

**Comment by ctarda** [comment]
FYI @koke

**Pull request merged**